src/localization/bundles/en.json

```diff
@@ -478,6 +480,15 @@
     "deploy.get.log.error": "Failed to get the log for playbook {0} - {1}. {2}",
     "deploy.progress.failure": "Cloud provisioning has failed. Please review the errors in the log, hit Back, address the issue by editing any necessary files, re-run Validate, and Deploy again.",
 
+    "update.progress.heading": "Cloud update deployment in progress",
+    "update.prepare.progress.heading": "Preparing model update deployment in progress",
+    "update.progress.failure": "Failed to deploy cloud update",
+    "update.deploy.confirm": "The deployment of the model update may disrupt your cloud.  Do you want to proceed?",
+    "update.model.heading": "Update model files",
+    "update.model.confirm.title": "Update model",
+    "update.model.confirm.msg": "This will run config-processor-run.yml and ready-deployment.yml playbooks for the model update. Check Run full deployment if want to deploy the model changes to the cloud. \n\nRunning prepare deployemnt playbooks or full deployment playbooks may cause inconsistency of the cloud. Do you want to proceed? ",
```

Comment:
I would change the first sentence to:
This will run the config-processor-run.yml and ready-deployment.yml playbooks to update the model.

I would change `Check Run full deployment if want to deploy the model changes to the cloud` to `Check "Run full deployment" to deploy the model changes to the cloud`

there's a misspelling later in the paragraph
`deployemnt` should be `deployment`